dist/tools/doccheck/check.sh

```diff
@@ -74,3 +75,40 @@ then
     echo "${UNDEFINED_GROUPS_PRINT}"
     exit 2
 fi
+
+# Check for groups defined multiple times:
+# - remove empty spaces from the beginning of the line
+# - replace spaces with ':' for easily looping on the result when printing
+# - lines starting with '1:' are excluded
+MULTIPLE_DEFINED_GROUPS=$(echo "${DEFINED_GROUPS}" | uniq -c | \
```

Comment:
Do they have to be counted? If not, you can simplify this operation to:

```suggestion
MULTIPLE_DEFINED_GROUPS=$(echo "${DEFINED_GROUPS}" | uniq -d)
```

From [specificaiton]:

```
-d
    Suppress the writing of lines that are not repeated in the input.
```